flake.nix

```diff
@@ -0,0 +1,72 @@
+/*
+   Copyright 2016-2017 SiFive, Inc.
```

Comment:
no copyright here. License is enough. The nix support was from plct team.

Reply:
Ok, that's a mistake 🤣